core/src/main/java/org/fao/geonet/kernel/AccessManager.java

```diff
@@ -23,18 +23,8 @@
 
 package org.fao.geonet.kernel;
 
-import static org.fao.geonet.repository.specification.OperationAllowedSpecs.hasMetadataId;
-import static org.fao.geonet.repository.specification.OperationAllowedSpecs.hasOperation;
-import static org.springframework.data.jpa.domain.Specifications.where;
-
-import java.sql.SQLException;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
-import java.util.StringTokenizer;
-
+import jeeves.server.UserSession;
```

Comment:
Is there any standards for sorting imports? Tends to add noise to PR reviews ...

Reply:
personally using intellij optimize import tools, I think it can be configured, I also know that some others are using eclipse.

Reply:
The following seems to indicate how to make Intellij use the same formatting as eclipse.
https://stackoverflow.com/questions/14716283/is-it-possible-for-intellij-to-organize-imports-the-same-way-as-in-eclipse

I believe it would end up being the following in the .editorconfig file.  
```
ij_java_class_count_to_use_import_on_demand = 99
ij_java_imports_layout = $*,|,java.**,|,javax.**,|,org.**,|,com.**,|,*
```

I have not tested to see if that would resolve this issue?

Reply:
file:///home/cmangeat/Bureau/import_settings.png

got something minimizing noise using such a settings, need to be perfected.